### ISSUE TITLE
Modify the "Join the project" default email text

### ISSUE THREAD

**Issue by innnkkki** [GENI-NSF/geni-portal#616]
When a user asks to join a project there is a default text sent along, the user can modify it
but I doubt that most people will. 

Today I got a join request from joe Membretti and the text says:
"
May I join GENI project 'GeniExperimentEngine'?
I think I need to do GENI research in your project.
I am a student in your lab.
To handle my request, go to the GENI Portal here:
https://portal.geni.net/secure/handle-project-request.php?project_id=3236e07c-587d-411d-83e7-3927dfeb1ff6&member_id=1571ad67-de17-4e65-ab08-223098b14620&request_id=488

Thank you,
Joe J Mambretti
"

Joe is clearly not a student so we should rephrase it to be more general. 

An easy fix is to just remove the the sentence "I am a student in your lab."

 _Imported from trac ticket [#616](http://trac.gpolab.bbn.com/proto-ch/ticket/616),  created by nriga on 06-13-2013 at 10:42, last modified: 06-13-2013 at 15:29_

**Comment by ahelsing** [comment]
Fixed.

_Trac comment by bujcich on 06-13-2013 at 15:29_
